client/src/components/EditProfile/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,33 @@
+import { createStyles, makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      '& > *': {
+        margin: theme.spacing(1),
+      },
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
```

Comment:
Alternatively, you can add these styles directly to a `Box` component through its props ([docs](https://v4.mui.com/system/basics/#all-inclusive)).